src/main/java/com/game/Battleship/controllers/BoardController.java

```diff
@@ -1,17 +1,52 @@
 package com.game.Battleship.controllers;
-
 import com.game.Battleship.models.Board;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RestController;
+import com.game.Battleship.models.Ship;
+import com.game.Battleship.repository.BoardRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+import java.util.Map;
 
 @RestController
 public class BoardController {
 
-    @GetMapping("/board")
-    public Board getBoard(){
+    @Autowired
+    private BoardRepository boardRepository;
+
+    @GetMapping("/boards")
+    public List<Board> getAllBoards(){
+
+        return boardRepository.allBoards();
+
+    }
+
+    //json object example: {"row": 0, "column": 1, "shipType": "Cruiser", "shipSize": 3, "isHorizontal": false }
+    @PatchMapping(value = "/boards/place")
+    public Board placeShip(@RequestBody Map<String, Object> changes) {
```

Comment:
Missing test.

Reply:
Is this tested by placeShipHorizontal & placeShipVertical? If so then the naming doesn't indicate.

Reply:
I didn't write the tests just yet for the patch request.

Reply:
Done!